src/helpers/blood-pressure-data-providers/query-blood-pressure.ts

```diff
@@ -0,0 +1,26 @@
+import surveyBloodPressureDataProvider, { BloodPressureDataParameters as SurveyBloodPressureDataParameters } from "./survey-blood-pressure-data-provider";
+
+export interface BloodPressureDataPoint {
+    dateLabel: string,
```

Comment:
Remove dateLabel here and just compute it when rendering

Reply:
I did remove this as a field, and am doing the format not on the dumbbell render, but on the creation of the xaxis text for the dumbbell